content/workshops/express-js/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+date: 2019-10-04
+title: Express JS
+Ready: True
+weight: 15
```

Comment:
ditto n the frontmatter

Reply:
edited, should be fine now changed it to little r and little t